docs/introduction.md

```diff
@@ -6,12 +6,12 @@ sidebar_position: 1
 slug: /
 ---
 
-Welcome fellow dreamer to the documentation of Pixelorama, your free and open source pixel-art creation tool, developed by Orama Interactive and our contributors. If you are new to this documentation, we recommend you start by learning [how to install the application](user_manual/installation), and then read the [User Interface Basics](user_manual/user_interface/user_interface_basics) page to familiarize yourself with Pixelorama's workspace. The ultimate goal of this program is to be able to catch up with the user's imagination and assist them in creating any kind of pixel art they wish. This could be a cool landscape, a dope animation, game graphics, random sketches or a well thought pixelated meme. Thank you for coming this far into researching our tool!
+Welcome, fellow dreamer, to the documentation of Pixelorama: your free and open source pixel-art creation tool, developed by Orama Interactive and our contributors. If you are new to this documentation, we recommend you start by learning [how to install the application](user_manual/installation), then reading the [User Interface Basics](user_manual/user_interface/user_interface_basics) page to familiarize yourself with Pixelorama's workspace. The ultimate goal of this program is to be able to keep up with the user's imagination and assist them in creating any kind of pixel art they wish. This could be a cool landscape, a dope animation, game graphics, random sketches or a well thought pixelated meme. Thank you forresearching our tool!
```

Comment:
Shouldn't "forresearching" be "for researching"?

Reply:
Your right, thanks. I edited the file to fix it.